openlcs/libs/unpack.py

```diff
@@ -17,6 +19,27 @@ def get_mime_type(filepath):
     return mime_type
 
 
+def create_dir(directory):
+    """
+    Create a directory to store non source RPMs files.
+    """
+    try:
+        if os.path.exists(directory):
+            shutil.rmtree(directory, ignore_errors=True)
+        os.makedirs(directory)
+    except Exception as e:
+        raise RuntimeError(e) from None
+    return directory
+
+
+def run_cli(cmd):
+    try:
+        subprocess.call(cmd, shell=True)
+    except subprocess.CalledProcessError:
+        error = f'Failed to run cmd {cmd}'
+        return '', error
```

Comment:
why would you want to return a message like this, is there a special use case? I didn't see anywhere this `run_cli` is called.

Reply:
Deleted this function that was designed before the function uncompress_tar_file.